its/plugin/tests/pom.xml

```diff
@@ -99,105 +102,53 @@
         </plugin>
       </plugins>
     </pluginManagement>
+    <plugins>
+      <plugin>
+        <groupId>com.googlecode.maven-download-plugin</groupId>
+        <artifactId>download-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>unpack-jre</id>
+            <phase>generate-test-resources</phase>
+            <goals>
+              <goal>wget</goal>
+            </goals>
+            <configuration>
+              <url>${openjdk8.url}</url>
+              <unpack>true</unpack>
+              <outputDirectory>${project.build.directory}/jre</outputDirectory>
+              <sha256>${openjdk8.sha256}</sha256>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
   </build>
 
   <profiles>
     <profile>
-      <id>download-jre-8-linux</id>
-      <activation>
-        <os>
-          <family>unix</family>
```

Comment:
Unfortunately, `unix` is active on linux and mac. The linux filter should be `<family>unix</family><name>Linux</name>`.